fiasco/tests/test_ion.py

```diff
@@ -47,12 +47,16 @@ def test_scalar_temperature(hdf5_dbase_root):
     ion = fiasco.Ion('H 1', 1 * u.MK, hdf5_dbase_root=hdf5_dbase_root)
     ioneq = ion.ioneq
     assert ioneq.shape == (1,)
+    assert np.isnan(ioneq)
```

Comment:
You writing this test has already caught a bug!!!! This should not be a NaN! PR coming soon! I'll rebase and fix this value.

Reply:
In this case, we can actually test approximately for correctness as `ioneq` is just interpreting from the supplied data.